fix(query-devtools): Update default devtools button position to bottom-right 🌱

agustints opened the pull request:
## Summary
This pull request addresses a comment from the maintainers regarding the default position of the query devtools. The change aligns the default devtools position with the bottom-right corner to maintain consistency between adapters and documentation.

## Changes Made
Updated the default position of the query devtools to bottom-right.
Adjusted relevant documentation to reflect the updated default position.

## Context
The maintainers suggested aligning the default devtools position with the bottom-right corner to maintain consistency across adapters and documentation. This adjustment ensures a unified experience for users and developers using the query library.

## Related Issues
Closes #6242
<img width="960" alt="Screenshot 2024-01-16 at 16 14 36" src="https://github.com/TanStack/query/assets/20648944/a43572a0-66ea-440c-aaf9-db2d19467ad3">

arnoud-dv commented:
I was just working on the Angular devtools and fixed this in there. Sorry for the conflict.

Noticed that default seems to have changed for position as well: it is mentioned the default is `bottom` but it's shown at the right when nothing is set.

agustints commented:
Hi @arnoud-dv, awesome thanks for fixing it for Angular, I updated the branch and fixed the conflicts 👌 
Also checked the `position` attribute and it's working well 👍
